.gitignore

```diff
@@ -18,3 +18,4 @@ webpack.report.html
 .graphqlconfig
 schema.json
 .yardoc
+.rspec-failures
```

Comment:
I usually run the test from within the folder, ie `cd decidim-proposals && bundle exec rspec`. 

What happens in this case? Should we add `**/.rspec-failures` to the `.gitignore` file?

Reply:
It seems to just work. At least that's what I do too and I don't see the file.

Reply:
OK then!